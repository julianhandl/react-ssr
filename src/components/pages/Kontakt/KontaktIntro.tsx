import React from 'react';
import './KontaktIntroStyles.scss';

const KontaktIntro : React.FunctionComponent = () => {
    return (
        <div className="container">
            <div className="kontakt-intro">
                <div className="kontakt-intro__slogan">
                    <h1>
                        <span>Kontakt</span>
                    </h1>
                    <p>
                        <span>User Shopsystem ist noch in den Kinderschuhen. </span>
                        <span>Wir beraten Sie und schreiben Ihnen gerne ein Angebot </span>
                        <span>solange unsere Webseite noch Laufen lernt.</span>
                    </p>
                </div>
            </div>
        </div>
    );
};

export default KontaktIntro;