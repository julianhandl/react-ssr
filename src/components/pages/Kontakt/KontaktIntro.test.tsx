import React from "react";
import { mount } from "enzyme";
import KontaktIntro from "./KontaktIntro";

describe("KontaktIntro", () => {
    it("should render normal", () => {
        const wrapper = mount(<KontaktIntro />);
        expect(wrapper).toMatchSnapshot();
    })
})