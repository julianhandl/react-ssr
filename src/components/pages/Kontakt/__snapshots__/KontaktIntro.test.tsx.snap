// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KontaktIntro should render normal 1`] = `
<KontaktIntro>
  <div
    className="container"
  >
    <div
      className="kontakt-intro"
    >
      <div
        className="kontakt-intro__slogan"
      >
        <h1>
          <span>
            Kontakt
          </span>
        </h1>
        <p>
          <span>
            User Shopsystem ist noch in den Kinderschuhen. 
          </span>
          <span>
            Wir beraten Sie und schreiben Ihnen gerne ein Angebot 
          </span>
          <span>
            solange unsere Webseite noch Laufen lernt.
          </span>
        </p>
      </div>
    </div>
  </div>
</KontaktIntro>
`;
